cvat/apps/engine/tests/utils.py

```diff
@@ -455,3 +456,33 @@ def filter_dict(
     d: dict[str, Any], *, keep: Sequence[str] = None, drop: Sequence[str] = None
 ) -> dict[str, Any]:
     return {k: v for k, v in d.items() if (not keep or k in keep) and (not drop or k not in drop)}
+
+
+def compare_objects(self, obj1, obj2, ignore_keys, fp_tolerance=0.001, current_key=None):
+    key_info = "{}: ".format(current_key or "")
```

Comment:
This doesn't seem right - it'll set `key_info` to `: ` if `current_key` is `None`.